experiments/list_unsupported_report.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+
+# Include configuration variables.
+. "$(dirname $0)/build_vars.cfg"
+
+# The script reads two input files, one for standard output and one
+# for standard input. Then concatenates them into the same file. This
+# is done to avoid interleaved text between the two streams breaking
+# the parsing done.
+STDOUT=$1
+STDERR=$2
+file_name="$(cat $STDOUT $STDERR)"
+
+# Need to use ${spec_ext} and ${body_ext} inside some regex's here,
+# so add any quoting necessary
+quoted_spec_ext=$(printf "%s" "${spec_ext}" | sed 's/\./\\./g')
+quoted_body_ext=$(printf "%s" "${body_ext}" | sed 's/\./\\./g')
+# This redacting system is really pretty crude...
+sed '/^\[/ d' < "$file_name".txt | \
+   sed 's/"[^"][^"]*"/"REDACTED"/g' | \
+      sed "s/[^ ][^ ]*\.${quoted_body_ext}/REDACTED.${body_ext}/g" | \
+         sed "s/[^ ][^ ]*\.${quoted_spec_ext}/REDACTED.${spec_ext}/g" \
+   > "$file_name"_redacted.txt
+
+# Collate and summarise unsupported features
+LC_ALL=posix ./CollectUnsupported "$file_name".txt
```

Comment:
For the call to `CollectUnsupported` you might be best either modifying the C++ program to accept input from stdin, or cat the two stdout/stderr files into a single temporary file that you pass to the program.